wsagent/che-datasource-shared/src/main/java/org/eclipse/che/datasource/shared/exception/DatabaseDefinitionException.java

```diff
@@ -0,0 +1,32 @@
+package org.eclipse.che.datasource.shared.exception;
+
+/**
+ * Created by test on 7/15/17.
+ */
+public class DatabaseDefinitionException extends Exception{
+    /** Serialization UID. */
+    private static final long serialVersionUID = 1L;
+
+    public DatabaseDefinitionException() {
+    }
+
+
+    public DatabaseDefinitionException(final String message) {
```

Comment:
Is that `final` word really necessary there? I'm asking because I've noticed that there are many places where you don't use it and in this case you used it for an unknown for me reason.